tests/LANraragi/Plugin/Metadata/CopyArchiveTags.t

```diff
@@ -0,0 +1,224 @@
+use strict;
```

Comment:
Nice; but this is missing the "using namespace" of the new plugin in the global test file

Reply:
Are you referring to `modules.t`?

Reply:
yep, ty